actions/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@cowprotocol/contracts": "^1.4.0",
+    "@cowprotocol/cow-sdk": "^2.4.0-rc.0",
```

Comment:
Added the SDK, as @mfw78 pointed out, it should be ideally the source of truth for JS logic